Make R6 apis consistent (#653)
rstudio/plumber: NEWS.md

### Breaking changes

> * `plumb()` now returns an object of class `"Plumber"` (previously `"plumber"`). To check is the return value is a Plumber router, use new method `is_plumber()`. (#653)

Reviewer: ```suggestion
* `plumb()` now returns an object of class `"Plumber"` (previously `"plumber"`). To check if an object is a Plumber router, use new method `is_plumber()`. (#653)
```

### Breaking changes

* When `plumb()`ing a file (or `Plumber$new(file)`), the working directory is set to the file's directory before parsing the file. When running the Plumber API, the working directory will be set to file's directory before running.(#631)

> 

Reviewer: Let's also add a comment about `plumb()` now returning an object of class `Plumber` (not `plumber`), but you can now use `is_plumber()` if you need to check the class.
